src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.ts

```diff
@@ -124,7 +124,7 @@ export class DeploymentsDonutChartComponent implements AfterViewInit, OnChanges,
 
   ngOnDestroy(): void {
     if (this.chart) {
-      this.chart = this.chart.destroy();
+      this.chart.unload({done: () => this.chart = this.chart.destroy() });
```

Comment:
Spacing around/in the curly braces is inconsistent. OK otherwise.

Reply:
My bad and nice catch; it should be fixed now.